src/services/songs/getLikedSongs.ts

```diff
@@ -0,0 +1,35 @@
+import { Reaction } from "../../entities/reaction"
+
+type getLikedSongsInput = {
+    roomId?: string,
+    perPage: number,
+    page: number,
+    userId: string
+}
+
+export async function getLikedSongsData(input: getLikedSongsInput) {
+    const {
+        roomId,
+        perPage,
+        page,
+        userId,
+    } = input
+    let queryBuilder = Reaction.createQueryBuilder("reaction")
+    if (roomId !== undefined) {
+        queryBuilder.where(`reaction.searchkey like '%${roomId}%'`);
+    }
+    queryBuilder.andWhere(`reaction.searchkey like '%${userId}%'`)
+    queryBuilder.orderBy("reaction.created_time", "DESC")
```

Comment:
Order by latest created song? or song created at first?

Reply:
the recently created song should come first.